src/components/Mutations/MutationsTableComponent.vue

```diff
@@ -8,25 +8,27 @@
       lazy
       @page="onPage($event)"
       :totalRecords="totalRecords"
+      :rowClass="getOtherTransaction"
     >
       <Column field="moment" style="width: 30%" :header="$t('transactions.when')">
         <template #body v-if="isLoading">
-          <Skeleton class="w-6 my-1 h-1rem surface-300"/>
+          <Skeleton class="w-6 my-1 h-1rem surface-300" />
         </template>
         <template #body="mutation" v-else>
           <span class="hidden sm:block">{{ mutation.data.moment.toDateString() }}</span>
-          <span class="sm:hidden">{{
-            mutation.data.moment.toLocaleDateString('nl-NL', {
-              dateStyle: 'short'
-            })
+          <span class="sm:hidden"
+            >{{
+              mutation.data.moment.toLocaleDateString('nl-NL', {
+                dateStyle: 'short'
+              })
```

Comment:
Just a FYI, this will likely cause conflicts with GH-179 when merging due to the formatting changes.

Reply:
True, although I'll fix that when rebasing.